components/common-go/tracing/tracing.go

```diff
@@ -44,6 +45,12 @@ func Init(serviceName string, opts ...Option) io.Closer {
 		return nil
 	}
 
+	version := os.Getenv("VERSION")
```

Comment:
Again, this will not work with the installer. We no longer put a generic `VERSION` env var in place.
Instead, for the Go components you'll want to rely on the version that gets [baked into the binary](github.com/gitpod-io/gitpod/pull/6624). To that end, the services need to pass their version into the `Init` call.

Reply:
Can we add the VERSION to the installer? I'm afraid that the version baked into the binary differs between components, we were expecting to have the same `main.xxxx` into all of them 🤔

Reply:
How would this work for the TypeScript services @csweichel?